runtime/ftplugin/hlsplaylist.vim

```diff
@@ -0,0 +1,34 @@
+" Vim filetype plugin
+" Language:     M3U Playlist
+" Maintainer:	AvidSeeker <avidseeker7@protonmail.com>
+" Last Change:  2024-07-06
+"
+
+if exists("b:did_ftplugin")
+  finish
+endif
+let g:did_ftplugin = 1
+
+setlocal commentstring=#%s
```

Comment:
missing space for 'commentstring'

Reply:
missing `b:undo_ftplugin`

Reply:
> missing space for 'commentstring'

I thought about this. Directives `#EXTM3U` don't have a space. So I removed the space to follow this convention.

Reply:
I guess that is fine then.